use std::num::ParseIntError;

// Implementiraj 'multiply' sa ?
// Nemoj koristiti 'unwrap' ovde
fn multiply(n1_str: &str, n2_str: &str) -> __ {
}

fn main() {
    assert_eq!(multiply("3", "4").unwrap(), 12);
    println!("Success!");
}
